### PR TITLE
libirecovery: add

### PR DESCRIPTION
**Pull request by neheb**
Allows communicating with iOS devices in recovery mode.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

**Comment by Biswa96** [comment]
Change the maintainer name with your as you are the creator of PKGBUILD.

**Comment by neheb** [comment]
copy/pasted from libimobiledevice. Fixed.

File: mingw-w64-libirecovery/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.0.0
+pkgrel=1
+pkgdesc='The libirecovery library allows communication with iBoot/iBSS of iOS devices via USB (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
+url='https://www.libimobiledevice.org/'
+license=('GPL2+')
+options=('staticlibs')
```
Comment (neheb) [inline mingw-w64-libirecovery/PKGBUILD:13]: Curious what this does.

Comment (Biswa96) [reply mingw-w64-libirecovery/PKGBUILD:13]: "Leave static library (.a) files in packages." More here https://man.archlinux.org/man/PKGBUILD.5

Comment (neheb) [reply mingw-w64-libirecovery/PKGBUILD:13]: weird

**Pull request merged**